internal/lookoutingesterv2/lookoutdb/insertion_test.go

```diff
@@ -593,6 +595,54 @@ func TestUpdateJobRunsScalar(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestCreateJobErrorsBatch(t *testing.T) {
+	err := lookout.WithLookoutDb(func(db *pgxpool.Pool) error {
+		ldb := NewLookoutDb(db, fatalErrors, m, 10)
+
+		// Insert
+		err := ldb.CreateJobErrorsBatch(armadacontext.Background(), defaultInstructionSet().JobErrorsToCreate)
+		assert.Nil(t, err)
+		jobError := getJobError(t, db, jobIdString)
+		assert.Equal(t, expectedJobError, jobError)
+
+		// Insert again and test that it's idempotent
+		err = ldb.CreateJobErrorsBatch(armadacontext.Background(), defaultInstructionSet().JobErrorsToCreate)
+		assert.Nil(t, err)
+		jobError = getJobError(t, db, jobIdString)
+		assert.Equal(t, expectedJobError, jobError)
+
+		// If a row is bad then we should return an error and no updates should happen
+		_, err = ldb.db.Exec(armadacontext.Background(), "DELETE FROM job_error")
```

Comment:
Would it be worth check that if you try to update it with a different value, it keeps the original value?